layers/+spacemacs/spacemacs-defaults/keybindings.el

```diff
@@ -359,6 +362,11 @@
   :mode font-lock-mode
   :documentation "Toggle syntax highlighting."
   :evil-leader "ths")
+(spacemacs|add-toggle spacemacs/toggle-column-indexing
+  :on (spacemacs/toggle-column-indexing)
+  :off (spacemacs/toggle-column-indexing)
+  :evil-leader "tz")
```

Comment:
I think you want something like the following:
```emacs-lisp
(spacemacs|add-toggle column-indexing
  :status column-number-indicator-zero-based
  :on (setq column-number-indicator-zero-based t)
  :off (setq column-number-indicator-zero-based nil))
```
...Wait, what does this have to do with the rest of the PR?

Reply:
Emacs 26 introduced this functionality, and I want my functions to respect the setting. So in case you figured out you did not copy the path with the appropriate column number you can use this toggle to fix it up.